ocaml/xcp-rrdd/bin/rrdd/xcp_rrdd.ml

```diff
@@ -75,6 +75,7 @@ let start (xmlrpc_path, http_fwd_path) process =
   let server = Http_svr.Server.empty () in
   let open Rrdd_http_handler in
   Http_svr.Server.add_handler server Http.Post "/" (xmlrpc_handler process) ;
+  Http_svr.Server.add_handler server Http.Post "/RPC2" (xmlrpc_handler process) ;
```

Comment:
I think this is the wrong place. This is a handler for the xcp-rrdd daemon, not xapi. You should add it to `common_http_handlers` in `xapi.ml` instead.